### PR TITLE
[#157387794] Improve onboarding PIN screen

### PR DESCRIPTION
**Pull request by ClaudioConversa**
Improve PIN set during onboarding and set autofocus on pin input

screen shot 2018-06-04 at 14 54 48

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#157387794](https://www.pivotaltracker.com/story/show/157387794): Aggiornare lo schermo di inserimento del PIN durante l'onboarding

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/screens/onboarding/PinScreen.tsx [hunk 1]
```diff
@@ -189,39 +190,50 @@ class PinScreen extends React.Component<Props, State> {
...
       </Content>
     );
   }
+  // Render the description for the different states
+  public renderDescription(pinState: PinState) {
+    if (pinState.state === "PinUnselected") {
+      return (
+        <React.Fragment>
+          <Text>{I18n.t("onboarding.pin.pinInfo")}</Text>
+          <Text link={true}>{I18n.t("onboarding.pin.moreLinkText")}</Text>
```
[the remark this answers is not in the document]
Comment (cloudify) [reply ts/screens/onboarding/PinScreen.tsx]: as discussed today, please remove this link for now

**Comment by cloudify** [comment]
@fpersico please review

File: ts/screens/onboarding/PinScreen.tsx [hunk 2]
```diff
@@ -189,39 +190,45 @@ class PinScreen extends React.Component<Props, State> {
...
-
         <View spacer={true} extralarge={true} />
-
-        <Text>{I18n.t("onboarding.pin.pinInfo")}</Text>
-        <Text link={true}>{I18n.t("onboarding.pin.moreLinkText")}</Text>
+        {this.renderDescription(pinState)}
       </Content>
     );
   }
+  // Render the description for the different states
```
Comment (fpersico) [inline ts/screens/onboarding/PinScreen.tsx]: Empty line missing

**Comment by cloudify** [comment]
@ClaudioConversa pls resolve the conflicts, then we can merge

File: ts/components/Pinpad.tsx [hunk 3]
```diff
@@ -8,6 +8,8 @@ import { PIN_LENGTH } from "../utils/constants";
 type OwnProps = {
```
Comment (cloudify) [inline ts/components/Pinpad.tsx]: wrap this type with `Readonly<>`

**Pull request merged**